packaging/technical-addon/CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Orca
+(For internal splunkers only)
+To use orca,
+0. Set up a venv `mkdir -p ~/.venvs; python3 -m venv ~/.venvs/orca`
```

Comment:
Nit: For formatting, the code samples might be rendered slightly more clearly if we make the, blockquotes with ```. This will render each code sample on its own line and make what's required of the user more clearly stand out,